src/odemis/acq/move.py

```diff
@@ -47,13 +48,16 @@
     SEM_IMAGING: "SEM IMAGING",
     FM_IMAGING: "FM IMAGING",
     GRID_1: "GRID 1",
-    GRID_2: "GRID 2"
+    GRID_2: "GRID 2",
+    THREE_BEAMS: "THREE BEAMS"
 }
 
 ATOL_LINEAR_POS = 100e-6  # m
 ATOL_ROTATION_POS = 1e-3  # rad (~0.5°)
 RTOL_PROGRESS = 0.3
-SCALING_FACTOR = 0.03 # m (based on fine tuning)
+SCALING_FACTOR = 0.03  # m (based on fine tuning)
+SAFETY_MARGIN_5DOF = 100e-6  # m
```

Comment:
I updated this number which added me as a author to all the commits. All the credits go to @lazem however.

Reply:
I made it this way because ATOL_LINEAR_POS is also 100e-6, so there won't be a difference between 3 beams and active in this case.